tests/draw_test.py

```diff
@@ -31,8 +31,13 @@ class TestBase(unittest.TestCase):
 
     def setUp(self):
         tests_dir = os.path.dirname(os.path.realpath(__file__))
-        self.tests_data_dir = os.path.abspath(os.path.join(tests_dir, './data'))
-        self.tests_blessed_images_dir = os.path.abspath(os.path.join(tests_dir, './blessed_images'))
+        self.tests_data_dir = os.path.abspath(os.path.join(
```

Comment:
Should we add a few lines on the README or in some comments to explain the setup for tests?
Maybe instead of the README we could add a more technical document for describing such things